### PR TITLE
DDF-3078 updated UI ingest documentation.

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

updates documentation about ingest to include ingest from Intrigue UI.

#### Who is reviewing it?
@mcalcote @vinamartin @Lambeaux @jrnorth @ahoffer 

#### Choose 2 committers to review/merge the PR.
@andrewkfiedler
@clockard
@coyotesqrl
@lessarderic
@pklinef
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

verify new content is accurate
builds successfully

#### What are the relevant tickets?
[DDF-3078](https://codice.atlassian.net/browse/DDF-3078)
#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/jdocs/content/_running/ingesting-contents.adoc [hunk 1]
```diff
@@ -38,7 +38,11 @@ Simply place the desired files in the monitored directory and it will be ingeste
 If, for any reason, the files cannot be ingested, they will be moved to an automatically created sub-folder named `.errors`.
 Optionally, ingested files can be automatically moved to a sub-folder called `.ingested`.
 
-==== Medium Complexity (for more than a few resources)
+====== User Interface Ingest
+
+Files and resources can also be ingested directly from <<_ui_ingest, ${catalog-ui}>>.
+
+===== Medium Complexity (for more than a few resources)
```
Comment (josephthweatt) [inline distribution/docs/src/main/jdocs/content/_running/ingesting-contents.adoc]: Does there need to be an extra '=' here (line 45)?

File: distribution/docs/src/main/resources/_contents/_quickstart/quickstart-ingesting-contents.adoc [hunk 2]
```diff
@@ -8,9 +8,11 @@ This is one way to ingest into the catalog, for a complete list of the different
 ==== Ingesting Sample Data
 
 . Download a sample valid https://codice.atlassian.net/wiki/download/attachments/1179756/geojson_valid.json?version=1&modificationDate=1368249436010&api=v2[GeoJson file here].
-. Navigate in the browser to the Search UI at ${secure_url}/search.
-. Click the *upload* icon in the upper right corner.
-. Select the file to upload.
+. Navigate in the browser to the ${catalog-ui} at ${secure_url}/search.
```
Comment (vinamartin) [inline distribution/docs/src/main/resources/_contents/_quickstart/quickstart-ingesting-contents.adoc]: Remove "the" before ${catalog-ui}

File: distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc [hunk 3]
```diff
@@ -89,6 +89,17 @@ In ${catalog-ui}, workspaces can be shared between users at different levels of
 .. To remove the workspace from users with specific roles, set the drop-down menu to *No Access* for that role.
 .. To remove individual users, remove their email from the email list.
 
+[[_ui_ingest]]
+=== Ingesting from the ${catalog-ui}
```
Comment (vinamartin) [inline distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc]: Remove "the"

File: distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc [hunk 4]
```diff
@@ -89,6 +89,17 @@ In ${catalog-ui}, workspaces can be shared between users at different levels of
...
+
+Resources can be ingested directly from ${catalog-ui}.
+
+. Navigate in the browser to the ${catalog-ui} at ${secure_url}/search.
```
Comment (vinamartin) [inline distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc]: Remove "the"

File: distribution/docs/src/main/jdocs/content/_running/ingesting-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_running/ingesting-contents.adoc]: Yes. It's pushing this section to be a lower level header.

File: distribution/docs/src/main/resources/_contents/_quickstart/quickstart-ingesting-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_quickstart/quickstart-ingesting-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc]: fixed.

**Comment by ricklarsen** [comment]
build now

**Pull request merged**